lib/orbital/api.rb

```diff
@@ -2,6 +2,8 @@ module Killbill #:nodoc:
   module Orbital #:nodoc:
     class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
+      FIVE_MINUTES_AGO = (1 * 300)
```

Comment:
Well, the raw messages we persisted already indicates that this is an `UNDEFINED` response. I can add a log for better tracking when at `to_transaction_plguin_info`; but not sure we would need to put that logic for each call when we return `trx plugin info`?